LICENSE

```diff
@@ -2,11 +2,11 @@ MIT License
 
 Copyright (c) 2016 Software Engineering Education - FOSS Resources
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
+Permission is hereby granted, free of charge, to any food obtaining a copy
```

Comment:
Should change back to "person". Same for Line 9.

Reply:
Good catch!

Reply:
Thanks!